blog/2022-09-12-psp-deprecation.mdx

```diff
@@ -0,0 +1,47 @@
+title: KubeSlice Security and PSP Deprecation
+description: This is my first post on Docusaurus 2.
+slug: welcome-docusaurus-v2-one
+authors:
+  - name: Leon Nunes
+    url: https://github.com/afro-coder
+hide_table_of_contents: false
+---
+
+In Kubeslice security is something that is Step 0 rather than an afterthought, today we will discuss about the security features Kubeslice provides.
+
+Kubernetes itself is a vast ecosystem, with its continuous growth there are things that are deprecated, [Pod Security Policies](https://kubernetes.io/docs/concepts/security/pod-security-policy/)(PSP) is one of those things.
+
+#### So what exactly is Pod Security Policies?
+> PodSecurityPolicy is a built-in [admission controller](https://kubernetes.io/blog/2019/03/21/a-guide-to-kubernetes-admission-controllers/) that allows a cluster administrator to control security-sensitive aspects of the Pod specification.
+One or more PodSecurityPolicy resources are created in a cluster to define the requirements Pods must meet. Then, RBAC rules are created to control which PodSecurityPolicy applies to a given pod. If a pod meets the requirements of its PSP, it will be admitted to the cluster as usual.
+
+**Note**: The [Kubernetes 1.25 release](https://kubernetes.io/blog/2022/08/04/upcoming-changes-in-kubernetes-1-25/) will remove PSP so if you rely on PSP you should follow the instructions for [migration to Pod Security Admission](https://kubernetes.io/docs/tasks/configure-pod-container/migrate-from-psp/)
+
+### How KubeSlice implements Security and PSS?
+KubeSlice segregates the Network in Slices as applications shouldn't need to communicate with all the resources in the cluster this reduces the blast radius, kubeslice creates slices, which is a collection of namespaces, network, applications over clusters or over different clouds.
+This prevents noisy/chatty applications trying to communicate in a cluster.
+
+Kubernetes defines a standard called [Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/)(PSS),
+>The Pod Security Standards define three different policies to broadly cover the security spectrum. These policies are cumulative and range from highly-permissive to highly-restrictive.
+
+This guide outlines the requirements of each policy.
+| Profile | Description
+| -------- | -------- |
+| Privileged | Unrestricted policy, providing the widest possible level of permissions. This policy allows for known privilege escalations.     |
+|Baseline|Minimally restrictive policy which prevents known privilege escalations. Allows the default (minimally specified) Pod configuration.|
+|Restricted|Heavily restricted policy, following current Pod hardening best practices.|
+
+These are Cluster and namespace specific.
+
+However, since Kubeslice creates slices which can span over clusters one can create a secure slice by setting the Pod Security Standards(PSS) for a set of namespaces, these namespaces will then reside inside a slice.
```

Comment:
```suggestion
However, since Kubeslice creates slices which can span over clusters one can create a secure slice 
by setting the Pod Security Standards(PSS) for a set of namespaces. These namespaces will then 
reside inside a slice.
```

Reply:
Done! Changes have been made accordingly